addon/components/file-renderer/component.js

```diff
@@ -24,10 +24,14 @@ export default Ember.Component.extend({
     width: '100%',
     height: '100%',
     allowfullscreen: true,
-    mfrUrl: Ember.computed('download', function() {
+    version: null,
+    mfrUrl: Ember.computed('download', 'version', function() {
         var base = config.OSF.renderUrl;
```

Comment:
This variable is only used once. Should probably just use `config.OSF.renderUrl` instead.